language-support/scala/bindings-akka/src/main/scala/com/digitalasset/ledger/client/binding/LedgerClientBinding.scala

```diff
@@ -99,7 +99,7 @@ class LedgerClientBinding(
         .map(_.map(compositeCommandAdapter.transform))
         .via(tracking)
 
-  private def createRetry[C](retryInfo: RetryInfo[C], completion: Completion): SubmitRequest = {
+  private def createRetry[C](retryInfo: RetryInfo[C], ignored: Any): SubmitRequest = {
```

Comment:
Is this for backwards compatibility? If so maybe useful to add a comment?

Reply:
It will become explicit in #6907 (it's to match a HOF signature), but I can't do that properly in this PR without a lot more mess.

Reply:
Makes sense, thanks for the explanation and also thanks for splitting up the changes!